toolkit/tools/imagecustomizerapi/storage_test.go

```diff
@@ -123,7 +123,7 @@ func TestStorageIsValidBadEspFsType(t *testing.T) {
 
 	err := storage.IsValid()
 	assert.Error(t, err)
-	assert.ErrorContains(t, err, "ESP partition must have 'fat32' filesystem type")
+	assert.ErrorContains(t, err, "ESP partition must have 'fat32' or 'vfat' filesystem type")
```

Comment:
Might be also worth adding a test case specifically for VFAT.

Reply:
This test actually is setting the partition to `ext4` to trigger this error. So, it doesn't make sense to create a second one for vfat.

However, I did add a no-error test that uses vfat.